### PR TITLE
README.md: separate CLI and packages documentation

### PR DESCRIPTION
**Pull request by Pixep**
The README is a bit confusing right now, as it mixes concepts from both the CLI and pluma packages. Some effort to reorganize it a bit better.

### CODE REVIEW

**Review by mjftw — DISMISSED** [review]
Great idea, and the new code example is much less rubbish.

File: README.md [hunk 1]
```diff
@@ -3,47 +3,15 @@
 ![Native installation and tests](https://github.com/Witekio/pluma-automation/workflows/Native%20installation%20and%20tests/badge.svg)
 ![Docker Image CI](https://github.com/Witekio/pluma-automation/workflows/Docker%20Image%20CI/badge.svg)
 
-Pluma Automation (formerly Automation Lab) is a tool created by [Witekio](https://www.witekio.com) to perform black box testing of embedded hardware; designed to be as lightweight as possible!
+Pluma Automation (formerly Automation Lab) is a tool created by [Witekio](https://www.witekio.com) to perform automated testing on embedded devices. It has no requirements on the device side, but was primarily design for Linux-based systems. This tools is can be used in two different ways:
```
Comment (mjftw) [inline README.md:6]: To make it really clear that it's very general purpose, you could rephrase this line a little.
"It was primarily designed for Linux-based devices, but there is no requirement for Linux and it should work with any device." Or something like that.
Basically to make it clear to people not running Linux on their target that the tool is still useful to them.

Comment (Pixep) [reply README.md:6]: Hmm... yeah that's true for device control, but not for the test suite and all. Or maybe we need to move our tests to a `testsuite.linux.memory` etc.

Comment (Pixep) [reply README.md]: Removed the "but was primarily design for Linux-based systems" altogether. This can be specified in specific testsuite tests directly.

**Review by mjftw — APPROVED** [review]

**Pull request merged**